daemons/compute/client-go/main.go

```diff
@@ -67,45 +73,60 @@ func main() {
 
 	c := pb.NewDataMoverClient(conn)
 
-	log.Printf("Creating request...")
-	createResponse, err := createRequest(ctx, c, *workflow, *namespace, *source, *destination, *dryrun)
-	if err != nil {
-		log.Fatalf("could not create data movement request: %v", err)
-	}
+	for i := 0; i < *count; i++ {
+		log.Printf("Creating request %d of %d...", i+1, *count)
 
-	if createResponse.GetStatus() == pb.DataMovementCreateResponse_CREATED {
-		log.Printf("Data movement request created: %s", createResponse.GetUid())
-	} else {
-		log.Fatal("Create request failed: ", createResponse.String())
-	}
+		createResponse, err := createRequest(ctx, c, *workflow, *namespace, *source, *destination, *dryrun)
+		if err != nil {
+			log.Fatalf("could not create data movement request: %v", err)
+		}
 
-	for {
-		statusResponse, err := getStatus(ctx, c, createResponse.GetUid(), *maxWaitTime)
-		if statusResponse.GetStatus() == pb.DataMovementStatusResponse_FAILED {
-			log.Fatalf("Data movement failed: %v", err)
+		if createResponse.GetStatus() == pb.DataMovementCreateResponse_CREATED {
+			log.Printf("Data movement request created: %s", createResponse.GetUid())
+		} else {
+			log.Fatal("Create request failed: ", createResponse.String())
 		}
 
-		log.Printf("Data movement status %s", statusResponse.String())
-		if statusResponse.GetState() == pb.DataMovementStatusResponse_COMPLETED {
-			break
+		// Wait for request to be completed
+		for {
+			statusResponse, err := getStatus(ctx, c, createResponse.GetUid(), *maxWaitTime)
+			if statusResponse.GetStatus() == pb.DataMovementStatusResponse_FAILED {
+				log.Fatalf("Data movement failed: %v", err)
+			}
+
+			log.Printf("Data movement status %s", statusResponse.String())
+			if statusResponse.GetState() == pb.DataMovementStatusResponse_COMPLETED {
+				break
+			}
+
+			time.Sleep(time.Second)
 		}
 
-		time.Sleep(time.Second)
+		log.Printf("Data movement completed: %s", createResponse.GetUid())
 	}
 
-	log.Printf("Data movement completed: %s", createResponse.GetUid())
+	// Get the list of Data Movement Requests
+	request_uids := getRequestsList(ctx, c, *workflow, *namespace, true)
+	log.Printf("List of Data movement requests: %v", strings.Join(request_uids, ", "))
```

Comment:
If you're using %v,  then just use bare request_uids as the arg, and it'll print out some readable representation of the list.  It's not a bad way to go.

If you're using strings.Join(), then you're  getting a string back and you can print that with %s.